packages/hadron-document/src/element.ts

```diff
@@ -736,7 +739,7 @@ export class Element extends EventEmitter {
         element.expand(expandChildren);
       }
     }
-    this.emit(ElementEvents.Expanded, this);
+    this._bubbleUp(ElementEvents.Expanded, this);
```

Comment:
Note: This is intentional and used in the virtualised list.   I forgot to remove this from the commit when I was splitting but I also think its harmless if it goes in main with this patch.